configs/11.0/packages/valgrind/sources

```diff
@@ -20,18 +20,18 @@
 #
 
 ATSRC_PACKAGE_NAME="Valgrind"
-ATSRC_PACKAGE_VER=3.11.0
+ATSRC_PACKAGE_VER=3.13.0
 # Don't update the Valgrind revision without updating the VEX revision too.
 # They have to be in sync.
-ATSRC_PACKAGE_REV=15905
-ATSRC_PACKAGE_VEX_REV=3223
+ATSRC_PACKAGE_REV=16446
+ATSRC_PACKAGE_VEX_REV=3385
```

Comment:
This misses one commit from the branch.

When I mentioned that if we kept checking out from trunk, we should checkout 3385, that was only valid if we actually kept checking out from trunk.

Now that we are checking out from the 3.13 branch, we have to get the revision that backports 3385 into the branch. The following command and output tells us that we should get revision 3386 in the branch:

```
$ svn log --stop-on-copy svn://svn.valgrind.org/vex/branches/VEX_3_13_BRANCH
------------------------------------------------------------------------
r3386 | sewardj | 2017-05-31 11:47:53 -0300 (Wed, 31 May 2017) | 2 lines

Merge, from trunk, r3385 (Handle x86 CET prefixes (32 bit only)).
```

So, you have to change ATSRC_PACKAGE_VEX_REV to 3386.

OK with this change.